packages/InteractiveIcon/Limited.jsx

```diff
@@ -11,8 +13,8 @@ import StyledInteractiveIconSVG from './shared/StyledInteractiveIconSVG'
 
 const iconSize = props =>
   handleResponsiveStyles({ size: props.size }, ({ size }) => ({
-    width: size === 20 ? '20px' : '24px',
-    height: size === 20 ? '20px' : '24px',
+    width: size === 20 ? '1.25rem' : '1.5rem',
+    height: size === 20 ? '1.25rem' : '1.5rem',
```

Comment:
Is 1.25rem and 1.5rem magic numbers or is this a TDS standard? I can't really tell here from looking at the code.

Reply:
1.25 rem and 1.5 rem are very common to see in TDS (see https://github.com/telus/tds-core/blob/master/packages/Box/Box.jsx#L9). However this could be another place where we could use something like a pixel-to-rem conversion from the `@tds/shared-utils` package.